restapi/api.go

```diff
@@ -6,20 +6,23 @@ import (
 	"github.com/go-chi/render"
 	"github.com/uwblueprint/shoe-project/restapi/rest"
 	"github.com/uwblueprint/shoe-project/server"
+	"gorm.io/gorm"
 )
 
 // namespace api
-type api struct{}
+type api struct {
+	database *gorm.DB
+}
 
-func Router() (http.Handler, error) {
+func Router(db *gorm.DB) (http.Handler, error) {
 	r := server.CreateRouter()
-	api := api{}
+	api := api{database: db}
```

Comment:
usually convention in go to write this line as:
```go
api := api{
    database: db,
}
```
In our case there is only one field so it doesn't matter